aws/main.tf

```diff
@@ -369,7 +405,12 @@ for i in $(seq 0 $(($${#private_ips[@]} - 1))); do
     echo "    connection_type: \"ssh\"" >> config.yml
     echo "    private_key: \"\"" >> config.yml
     echo "    ssh_common_args: \"\"" >> config.yml
-    echo "    roles: []" >> config.yml
+    if [ $(($${#private_ips[@]} % 2)) -eq 0 ] || [ $i -lt $(($${#private_ips[@]} - 1)) ]; then
```

Comment:
Can you put a comment for when this condition evaluates to true? Or just an explanation of which nodes get critical_services=true?

Reply:
yeah sure.